tron/utils/timeutils.py

```diff
@@ -93,7 +93,7 @@ def parse(cls, date_str, dt=None):
         based on date format (Ex: seconds for unixtime, days for day).
         """
         dt = dt or current_time()
-
+        date_str = date_str.replace(" ", "")
```

Comment:
this should probably be a regex replace for whitespace - otherwise, `'shortdate -  1' would still fail to parse

Reply:
hm, I'm not sure how 'shortdate - 1' would fail to parse with my logic. I try not to use regex since it's slower 😅

Reply:
ah - I was just mis-remembering how `replace()` worked :p (for some reason I had it in my head that it wouldn't replace multiple consecutive spaces)

Reply:
(in other words, my brain stops working after 5pm 🤣 )